middle_end/flambda2/terms/code_size.ml

```diff
@@ -120,14 +120,14 @@ let arith_conversion_size src dst =
 
 let unbox_number kind =
   match (kind : Flambda_kind.Boxable_number.t) with
-  | Naked_float -> 1 (* 1 load *)
+  | Naked_float | Naked_vec128 -> 1 (* 1 load *)
```

Comment:
I'm not completely sure that it's accurate to count 128-bit loads as a single instruction, but I don't think it matters that much since all these numbers are rough guesses anyway.

Reply:
Yeah, it depends on how the backend implements 128-bit loads, but it's a single instruction on amd64 and arm64.